pages/videos.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <v-container grid-list-md>
```

Comment:
are you using this: 'grid-list-md'? it is not a valid prop for v-container https://vuetifyjs.com/en/components/grids/#api